examples/cstar_blueprint_roms_marbl_example.yaml

```diff
@@ -11,8 +11,8 @@ components:
         source_repo: 'https://github.com/marbl-ecosys/MARBL.git'
         checkout_target: 'marbl0.45.0'
       additional_code:
-        source_repo: 'https://github.com/dafyddstephenson/roms_marbl_example.git'
-        checkout_target: 'cstar_latest'
+        source_repo: 'https://github.com/CWorthy-ocean/cstar_blueprint_roms_marbl_example'
+        checkout_target: '69ee87b554082c46e3f138a9b3faf17439966e90'
```

Comment:
Is this supposed to change? Shouldn't this still point to the latest version, we're just about to update the latest version?

Reply:
If this is fine then happy to merge this, with the understanding that we come back to it to solve #15.

Reply:
There's a conflict between pointing to the latest version (which has been regularly changing) and pointing to a fixed version hosted by [C]Worthy (in line with C-Star design philosophy). This commit hash at present points to the latest version, which I believe will be static for a minute. I'd prefer to verify that everything is still working internally and then update the target on the blueprint than just have the blueprint point to a fluid repo where changes could break things.

Reply:
That makes sense - but then presumably this line should point to either "main" or a tag, rather than a specific commit?

If this is intentional though we can merge this fix now and talk about this in a another issue.

Reply:
There's not a tag yet as I still anticipate a "first final version" once we have the proper BGC datasets locked in. `main` is not fixed. The commit hash is a temporary swap-in for what would be a tag.